#!/usr/bin/env python
PACKAGE = "pointcloud_filter"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("min_crop_height",     double_t,    0,     "Minimum crop height for the box filter",          1,    0,  20)
gen.add("max_crop_height",    double_t,    0,     "Maximum crop height for the box filter",          1,    0,   20)

exit(gen.generate(PACKAGE, "pointcloud_filter", "WallDetectionParameters"))